lambda/src/main/resources/db/migration/V31__add_consignment_status_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE "ConsignmentStatus"
+(
+    "ConsignmentStatusId" uuid NOT NULL,
+    "ConsignmentId" uuid NOT NULL,
+    "StatusType" text NOT NULL,
+    "Value" text NOT NULL,
+    "DateTime" timestamp with time zone not null,
+    CONSTRAINT "ConsignmentStatus_pkey" PRIMARY KEY ("ConsignmentStatusId"),
+    CONSTRAINT "ConsignmentStatus_Consignment_fkey" FOREIGN KEY ("ConsignmentId")
+    REFERENCES "Consignment" ("ConsignmentId") MATCH SIMPLE
+    ON UPDATE NO ACTION
+    ON DELETE NO ACTION
+    NOT VALID
```

Comment:
Is there a specific situation that this line is trying to handle?

I wouldn't have expected it to make a difference because this table will be empty when we create it, so there won't be any constraints that aren't handled. I see it's in some of the other scripts, but I don't think it should be in those either! If it doesn't do anything, I'd prefer to delete it so that it doesn't set a bad precedent for when we're modifying tables.

Reply:
I think I was trying to reflect how we've got the other tables set up. I'll remove it 👍

Reply:
Is it just the `NOT VALID` that I should remove or should I take out the `ON UPDATE NO ACTION` and `ON DELETE NO ACTION` as well?

Reply:
My preference would be to not include them, because it makes them look significant when they're really just the default. But because they are the default, it's not a big deal. Happy to approve the PR either way. 🙂

Reply:
I'll remove it since we want to go that direction in future anyway 👍